docs/reference/filebeat/filebeat-input-azure-eventhub.md

```diff
@@ -42,12 +61,108 @@ The name of the eventhub users would like to read from, field required.
 Optional, we recommend using a dedicated consumer group for the azure input. Reusing consumer groups among non-related consumers can cause unexpected behavior and possibly lost events.
 
 
+## Authentication [_authentication]
+
+The azure-eventhub input supports multiple authentication methods. The `auth_type` configuration option controls the authentication method used for both Event Hub and Storage Account.
+
+### Authentication Types
+
+The following authentication types are supported:
+
+- **`connection_string`** (default if `auth_type` is not specified): Uses Azure Event Hubs and Storage Account connection strings
+- **`client_secret`**: Uses Azure Active Directory service principal with client secret credentials
```

Comment:
If I'm understanding this change correctly, `connection_string` as an auth type was available before 9.3.0 (even though the `auth_type` option was not available). If that's the case we can specify that `client_secret` is being added in 9.3.0.

```suggestion
- **`connection_string`** (default if `auth_type` is not specified): Uses Azure Event Hubs and Storage Account connection strings
- {applies_to}`stack: ga 9.3.0` **`client_secret`**: Uses Azure Active Directory service principal with client secret credentials
```